src/Migration/RecordTransformer.php

```diff
@@ -127,25 +127,21 @@ public function applyHandlers(Handler\Manager $handlerManager, Record $recordToH
     protected function copy(Record $from, Record $to)
     {
         $sourceDocumentName = $this->sourceDocument->getName();
-        $data = $from->getData();
         $sourceFields = $from->getFields();
-        $destinationFields = $to->getFields();
         foreach ($sourceFields as $key => $field) {
             if ($this->mapReader->isFieldIgnored($sourceDocumentName, $field, MapInterface::TYPE_SOURCE)) {
                 unset($sourceFields[$key]);
-                unset($data[$field]);
             }
         }
-        $diff = array_diff($sourceFields, $destinationFields);
-        foreach ($diff as $field) {
+
+        $data = [];
+        foreach ($sourceFields as $field) {
             if (!$this->mapReader->isFieldIgnored($sourceDocumentName, $field, MapInterface::TYPE_SOURCE)) {
```

Comment:
Do we need rechecking ignored field for the second time? I see 
>if ($this->mapReader->isFieldIgnored($sourceDocumentName, $field, MapInterface::TYPE_SOURCE)) {

the line 132 have already done it

Reply:
Hi @victor-v-rad ! Yeah, you're totally right. I also removed the usage of the unnecessary `$data` array and the second `foreach`.